Xamarin.Forms.Platform.UAP/Xamarin.Forms.Platform.UAP.csproj

```diff
@@ -101,13 +101,16 @@
       <Generator>MSBuild:Compile</Generator>
     </Page>
   </ItemGroup>
+  <ItemGroup>
```

Comment:
Can we do this without adding this dependency ?

Reply:
According to Microsoft docs https://docs.microsoft.com/en-us/windows/uwp/design/shell/tiles-and-notifications/send-local-toast

Reply:
I am not sure that notification is a right way to use for DisplayToast. Probably I should implement it the same way as for WPF

Reply:
I actually like the native notification, what do you think @samhouts  ?

Reply:
I am worried about the name of the method. if we decide to add ShowNotification method, it will be the same implementation, so the user won't know is it a notification or a snackbar. But I like Windows Notifications, as it is a native way to show notification, and it is customizable and has action buttons